### DISCUSSION TITLE
آموزش کار با نسخه پایتون

### DISCUSSION THREAD

**Discussion by mohamad15516** [MortezaBashsiz/CFScanner#501] [Q&A] [answered]
**سلام ببخشید واقعا از تمام دوستانی که زحمت میکشند از ته قلبم تشکر میکنم من تا چند وقت پیش ایپی ها رو با نسخه ویندوز اسکن میکردم و نسخه ویندوزی واقعا عالی بود اما دیگه دسترسی به کامپیوتر ندارم و با برنامه userland نسخه پایتون اسکنر رو ران کردم و حتی اسکن هم کرد اینجا به دوتا مشکل بر خوردم چون زیاد وارد نیستم یکی چجوری و با چه دستوری فایلی که ایپی های اسکن شده داخلش هست رو باز کنم و آیا مثل نسخه ویندوزی امکان اسکن راندوم بین کل آیپی ها هست و قابلیت همون skip که توی نسخه ویندوزی هست رو هم داره اگر هست با چه دستوری میتونم اجراش کنم ممنونم**

**Comment by tempookian** [comment]
درود. 

 ---
- آموزش پایه [این‌جا](https://github.com/MortezaBashsiz/CFScanner/discussions/505) قرار داده شد.
--
- با استفاده از این دستوراحتمالا بتونید نتایج اسکن رو به جایی منتقل کنید که file manager روی گوشی اندرویدیتون بهش دست‌رسی داشته باشه. به طور مثال می‌تونید توی پوشه دانلودها کپیش کنید:
```bash
cp ./results /sdcard/Download -r
```
در این مورد دوستان دیگر هم اگر تجربه موفقی داشتن خیلی عالی می‌شه اگر راهنمایی کنند.

---

نزدیک‌ترین قابلیتی که توی نسخه پایتون به ذهنم می‌رسه قابلیت اسکن تصادفی تعداد آیپی مشخص از هر سابنته. مثلا برای اسکن ۵۰ آیپی از هر سابنت:
```bash
cfscanner -r 50
```
یا اسکن ۲۰ درصد از آیپی‌های هر سابنت:
```bash
cfscanner -r 0.2
```

برای دیدن جزئیات بیشتر قسمت [گزینه‌ها](https://github.com/MortezaBashsiz/CFScanner/tree/main/python#arguments) رو ببینید.

**Comment by mohamad15516** [comment, marked as answer]
ممنون از کمکتون و راهنمایی شما اما این دستور شما برام کار نکرد اما با وارد شدن به  پوشه فایل result و بعد انتقال خود فایل داخل پوشه موفق به انجام این کار شدم 😂  با این دستور 

`cd /result`

و بعد با زدن اسم فایل حاوی ایپی ها با دستور زیر تونستم فایل رو توی پوشه دانلود انتقال بدم و بتونم استفاده کنم 

`cp 20230423_183849_result.csv /sdcard/Download`

به این شکل زیر برای دوستانی که مثل خودم وار نیستن ممنون از تمام تلاش شما و راهنمایی مفیدتون ❤


![photo_2023-04-23_22-40-01](https://user-images.githubusercontent.com/113285931/233860095-3d75ef06-dd6c-4f96-8472-d3f078a762e8.jpg)

**Reply by tempookian** [reply]
یه تغییر ریزی دادم احتمالا اگر بلافاصله بعد از اسکن دستوری که گذاشتم رو ران کنید کل نتایج رو براتون ببره توی پوشه دانلود.

**Reply by mohamad15516** [reply]
متاسفانه بعد از اسکن دوباره و زدن دستور جدید مجدد این ارور رو میده 


![photo_2023-04-24_00-23-59](https://user-images.githubusercontent.com/113285931/233865841-bb4c7d3b-8b5e-42ed-8e91-021496b21ff2.jpg)
